app/helpers/alchemy/pages_helper.rb

```diff
@@ -299,7 +299,7 @@ def render_title_tag(options={})
         separator: ""
       }
       options = default_options.merge(options)
-      %(<title>#{render_page_title(options)}</title>).html_safe
+      content_tag(:title, render_page_title(options).html_safe)
```

Comment:
I guess it would be better to return a html safe string from `render_page_title` helper instead?